ignition-rendering.rb

```diff
@@ -0,0 +1,88 @@
+class IgnitionRendering < Formula
+  desc "Common libraries for robotics applications. Rendering Library"
+  homepage "https://bitbucket.org/ignitionrobotics/ign-rendering"
+  url "https://bitbucket.org/ignitionrobotics/ign-rendering/get/ca7e2c8d7755131c9374c99a1df41c6eee415739.tar.gz"
+  version "0.0.0~20170721~35b9bae"
```

Comment:
the end of the version string should be part of the commit, and the date should be the date of the commit